app/pb_kits/playbook/pb_date_range_inline/docs/_date_range_inline_default.html.erb

```diff
@@ -1 +1,7 @@
-<%= pb_rails("date_range_inline", props: { start_date: Date.new(2019, 06, 18), end_date: Date.new(2020, 03, 20) }) %>
+<div>
+  <%= pb_rails("date_range_inline", props: { start_date: Date.new(2019, 06, 18), end_date: Date.new(2020, 03, 20), size: "xs" }) %>
+  <%= pb_rails("date_range_inline", props: { margin_top: "xs",start_date: Date.new(2019, 06, 18), end_date: Date.new(2020, 03, 20) }) %>
+
+  <%= pb_rails("date_range_inline", props: { margin_top: "md", start_date: Date.new(Date.current.year, 01, 15), end_date: Date.new(Date.current.year, 8, 15), size: "xs", icon: true }) %>
+  <%= pb_rails("date_range_inline", props: { margin_top: "xs",start_date: Date.new(Date.current.year, 01, 15), end_date: Date.new(Date.current.year, 8, 15), icon: true }) %>
```

Comment:
Is there a reason `margin_top` is being added in the doc examples? If this is just to create separation between the examples it may be better to use </br> tags.

Reply:
@megantrimble That's exactly it, I can try to look for examples of that elsewhere of how that should be done. Do you know of a good one to look at off-hand?

Reply:
You can use `</br>` tags between the examples. Sorry I mentioned that above but forgot to wrap it so it showed up correctly.